src/plugins/Sim.js

```diff
@@ -0,0 +1,303 @@
+'use strict';
```

Comment:
This is not just Sim it is baseband / RIL configuration. File should be name accordingly.